matlab/test/fullmathtest.m

```diff
@@ -0,0 +1,127 @@
+% MATLAB GPU FINUFFT library math test, both precisions, both devices,
+% many-vector option (ntrans).
+% For type 3 the space-bandwidth product chosen so FFT size roughly same as
+% that in the type 1 and 2 cases.
+% Short runtime (~1 sec).
+% Barnett 4/2/25.
+clear
+
+isign   = +1;     % sign of imaginary unit in exponential
+if exist('cufinufft')==3 && exist('canUseGPU') && canUseGPU() % .mex exists?
```

Comment:
out of curiosity where is `canUseGPU` defined? is a matlab function?

Reply:
https://www.mathworks.com/help/matlab/ref/canusegpu.html